exp/api/remote/remote_headers.go

```diff
@@ -94,6 +94,26 @@ const (
 	writtenExemplarsHeader  = "X-Prometheus-Remote-Write-Exemplars-Written"
 )
 
+// WriteResponse represents the response from the remote storage upon receiving a remote write request.
+type WriteResponse struct {
```

Comment:
I think there's room to make this easy to CREATE by creating helpers, methods and removing ability to unnecessary read it. 

But we can just place comment "TODO" and do it later, ok to merge without it.